README.md

```diff
@@ -48,12 +48,18 @@ Note that the access key and secret key provided here is authorized to perform _
 
 ### Step 3: Create MinIO Tenant
 
-We can create the Tenant now. Before that, please ensure you have requisite nodes and drives in place and relevant PVs are created. In below example, we ask MinIO Operator to create a Tenant with 4 nodes, 16 volumes, and 16 Ti total raw capacity (4 volumes of 1 Ti per node). This means you need to have 4 PVCs of 1 Ti each, per node, and total of 4 nodes, before attempting to create the MinIO Tenant.
+We can create the Tenant now. Before that, please ensure you have requisite nodes and drives in place and relevant PVs are created. In below example, we ask MinIO Operator to create a Tenant yaml with 4 nodes, 16 volumes, and 16 Ti total raw capacity (4 volumes of 1 Ti per node). This means you need to have 4 PVCs of 1 Ti each, per node, and total of 4 nodes, before attempting to create the MinIO Tenant.
 
 We recommend [direct CSI driver](https://github.com/minio/operator/blob/master/docs/using-direct-csi.md) to create PVs.
 
 ```sh
-kubectl minio tenant create --name tenant1 --secret tenant1-secret --servers 4 --volumes 16 --capacity 16Ti --namespace tenant1-ns --storage-class direct.csi.min.io
+kubectl minio tenant create-yaml --name tenant1 --secret tenant1-secret --servers 4 --volumes 16 --capacity 16Ti --namespace tenant1-ns --storage-class direct.csi.min.io > tenant.yaml
```

Comment:
Just to confirm - users *must* produce `yaml` , modify, and then apply? We no longer have a straight `tenant create` option?

Reply:
Also - since this appears to output to stdout, I guess a user could take `kubectl minio tenant create-yaml ... > kubectl apply -f` ? I should check if kubectl can take from stdin